test-lib.sh

```diff
@@ -1150,10 +1154,11 @@ ct_run_tests_from_testset() {
     if [ $TESTCASE_RESULT -eq 0 ]; then
       test_msg="[PASSED]"
     else
-      test_msg="[FAILED]"
-      if [ -z "$IGNORE_UNSTABLE_TESTS" ] || [ $is_unstable -eq 0 ]; then
+      if [ -z "${IGNORE_UNSTABLE_TESTS:-""}" ] || [ $is_unstable -eq 0 ]; then
+        test_msg="[FAILED]"
         TESTSUITE_RESULT=1
       fi
+      test_msg="[FAILED][UNSTABLE-IGNORED]"
```

Comment:
I like the idea but it seems to me that the new message is used even when `IGNORE_UNSTABLE_TESTS` is not defined. Actually, it does not matter what combination of the two variables you have, the new message is used every time a test fails.

Reply:
right, it seems, it was like this from the beginning. please rereview.